### PR TITLE
task 2

### PR DESCRIPTION
**Pull request by maycatsby**
link https://maycatsby.github.io/ui-internship-lwo2/HTML/task2/

### CODE REVIEW

File: HTML/task2/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel = "stylesheet" href = "style.css"/>    
+    <title>Task 2 - filter the colors</title>
+</head>
+<body>
+    <header><h1>Select colors</h1></header>
+    <main class = 'wrapper'>
```
Comment (tkachukivan) [inline HTML/task2/index.html]: you don't need to use spaces in front and after `=` also generally there is a rule that in js we use single quotes and in html double, but at least keep consistency as you use both of them and it is not very good!!!

File: HTML/task2/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+           <div class = 'wrapper__inputs__spans'>
+                <input type = "checkbox" id = 'red' />
+                <label class = "label--red" for = 'red'>red</label>
```
Comment (tkachukivan) [inline HTML/task2/index.html]: why `--` when just `-` is completely ok, same below

**Review by tkachukivan — APPROVED** [review]
It would be more logical if checkboxes was selected by default and when you uncheck one you don't show values connected with this checkbox

**Review by tsvetkovv — CHANGES_REQUESTED** [review]
Please take a look at Ivan's comments

File: HTML/task2/style.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    font-size: 20px;
+}
+input[type="checkbox"] {
+    display: none;
+}
+input:checked + label {
+    color: grey;
+}
+input[id="red"]:not(:checked) ~ .span-red {
+    display: none;
```
Comment (tsvetkovv) [inline HTML/task2/style.css:11]: Could you use one rule for 3 selectors? It reduces code duplication

File: HTML/task2/style.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
+.wrapper {
+    font-size: 20px;
+}
+input[type="checkbox"] {
+    display: none;
+}
+input:checked + label {
+    color: grey;
+}
+input[id="red"]:not(:checked) ~ .span-red {
```
Comment (tsvetkovv) [inline HTML/task2/style.css:10]: `input#red`?

**Review by tsvetkovv — APPROVED** [review]

**Pull request merged**